test.memo.stdout.x

```diff
@@ -0,0 +1 @@
+X
```

Comment:
this file shouldn't be added in your PR, probably added by mistake

Reply:
This is a mistake. I'll remove it.